pom.xml

```diff
@@ -34,6 +34,16 @@
       <artifactId>json</artifactId>
       <version>20200518</version>
     </dependency>
+    <dependency>
+      <groupId>org.jsoup</groupId>
+      <artifactId>jsoup</artifactId>
+      <version>1.11.3</version>
```

Comment:
Looks like an old version of jsoup (or at least Iris' PR has a more recent version).